torch/csrc/utils/python_dispatch.cpp

```diff
@@ -202,6 +202,17 @@ void initDispatchBindings(PyObject* module) {
     c10::Dispatcher::singleton().checkInvariants();
   });
 
+  m.def("_dispatch_has_kernel", [](const char* name) -> bool {
```

Comment:
nice this is exactly what I needed to disallow access to overloads added by jit

Reply:
@anjali411 Do you think we can do some more general cleanup here? Maybe version `overloads()` that only returns dispatcher overloads?